### PR TITLE
Fix installation recipe for freetype version 2.5.2.

### PR DESCRIPTION
**Pull request by gregvonkuster**

### CODE REVIEW

File: packages/package_freetype_2_5_2/tool_dependencies.xml [hunk 1]
```diff
@@ -12,14 +12,21 @@
                         <package name="libpng" version="1.6.7" />
                     </repository>
                 </action>
-                <action type="shell_command">export LDFLAGS="-Wl,-rpath,$LIBPNG_ROOT/lib" &amp;&amp; ./configure --prefix=$INSTALL_DIR</action>
-                <action type="make_install" />
+                <action type="shell_command">
+                    <![CDATA[
+                        export LDFLAGS="-Wl,-rpath,$LIBPNG_ROOT/lib" &&
```
Comment (mvdbeek) [inline packages/package_freetype_2_5_2/tool_dependencies.xml]: @gregvonkuster 
Could you check if we can also eliminate this export line?
Normally libpng sets the LD_LIBRARY_PATH.
Then we can replace the whole shell_command section with a single autoconf line.

**Comment by gregvonkuster** [comment]
@mvdbeek I tried removing the LDFLAGS setting and things still worked, so made the additional change.  Thanks!

**Comment by mvdbeek** [comment]
@gregvonkuster looks good. I'll do a quick check compiling R,
if this works I'll merge. Thanks!

**Comment by mvdbeek** [comment]
Okay, everything compiled successfully, I'm merging and uploading to the toolsheds.

**Comment by gregvonkuster** [comment]
Thanks @mvdbeek !

**Pull request merged**